nncf/automl/agent/ddpg/memory.py

```diff
@@ -0,0 +1,226 @@
+from __future__ import absolute_import
```

Comment:
Please add Copyright notices for all new files.
https://github.com/openvinotoolkit/nncf/blob/1e9f253c7a17068209078a10b9688be1a5fe61bb/nncf/quantization/precision_init/adjacent_quantizers.py#L1-L12